winui/Kanban/events.md

```diff
@@ -0,0 +1,762 @@
+            taskDetail.Category = "Open";
+            taskDetail.IndicatorColorKey = "High";
+            taskDetail.Tags = new List<string>() { "Bug Fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri("ms-appx:///Assets/Kanban/People_Circle1.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "WinUI Issue";
+            taskDetail.Id = "646";
+            taskDetail.Description = "AxisLabel cropped when rotating the axis label";
+            taskDetail.Category = "Open";
+            taskDetail.IndicatorColorKey = "Low";
+            taskDetail.Tags = new List<string>() { "Bug Fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle2.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "Kanban Feature";
+            taskDetail.Id = "25678";
+            taskDetail.Description = "Provide drag and drop support";
+            taskDetail.Category = "In Progress";
+            taskDetail.IndicatorColorKey = "Low";
+            taskDetail.Tags = new List<string>() { "New control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle3.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "New Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Dragging events support for Kanban";
+            taskDetail.Category = "Closed";
+            taskDetail.IndicatorColorKey = "Normal";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle4.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "WF Issue";
+            taskDetail.Id = "1254";
+            taskDetail.Description = "HorizontalAlignment for tooltip is not working";
+            taskDetail.Category = "Review";
+            taskDetail.IndicatorColorKey = "High";
+            taskDetail.Tags = new List<string>() { "Bug fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle5.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+            return taskDetails;
+        }
+
+        /// <summary>
+        /// Handles the event when a Kanban card is tapped.
+        /// </summary>
+        /// <param name="parameter"></param>
```

Comment:
/// <summary>
        /// Handles the event when a Kanban card is tapped.
        /// </summary>
        /// <param name="parameter">The object.</param>

Reply:
Added parm text.